webpack/stripes-module-parser.js

```diff
@@ -5,7 +5,7 @@ const StripesBuildError = require('./stripes-build-error');
 const logger = require('./logger')('stripesModuleParser');
 
 // These config keys do not get exported with type-specific config
-const TOP_LEVEL_ONLY = ['permissions', 'icons', 'okapiInterfaces', 'permissionSets', 'actsAs', 'type', 'hasSettings'];
+const TOP_LEVEL_ONLY = ['permissions', 'icons', 'permissionSets', 'actsAs', 'type', 'hasSettings'];
```

Comment:
@skomorokh, we used to [remove only `type`](https://github.com/folio-org/stripes-core/blob/4fccc75bfbfd10153b55cf8335a0133862261260/webpack/stripes-module-parser.js#L66). We did we add all these other values?

Reply:
As mentioned in my PR and in the changelog, I attempted to keep things in one place so that eg. you didn't access icons or permissions via `modules.app.foo` as those are consumed elsewhere and made available through other means that should be used instead. Plus there'd be redundant copies now that the metadata not excluded there is placed in the array for each type.

The `okapiInterfaces` I'd considered as being used to generate module descriptors and didn't take into account the about page, total oversight on my part, apologies for the hassle.